src/views/courseDetail.pug

```diff
@@ -142,7 +142,7 @@ block content
           .image.text-center
             img.img-fluid.mt-2(src='/img/course-1.jpg' alt='')
           .buttons
-            if paidCourse
+            if user.role === 'professor' || paidCourse
```

Comment:
Về bản chất professor cũng là 1 user, nên chỗ này vì sao cần check logic như này em nhỉ?

Reply:
vì như nãy em có nói là em muốn cho professor có thể enroll vào course lập tức, bỏ qua các bước mua hay add to cart á chị

Reply:
@thanhbao922003